### ISSUE TITLE
Support for "minified" app/pinned tabs

### ISSUE THREAD

**Issue by tbs** [Croydon/vertical-tabs-reloaded#20]
Is it possible? Similar to what Tree Style Tabs does.

Thanks

**Comment by Croydon** [comment]
@tbs Do you mean a feature that allows grouping tabs together?

**Comment by Keith94** [comment]
He's asking to make pinned tabs be displayed along the top like so:

![snag 2016 03 26 18 48 03](https://cloud.githubusercontent.com/assets/5490615/14063428/5fa61ed0-f385-11e5-9264-e235d1dca5da.png)

**Comment by yafp** [comment]
To be honest ... displaying pinned tabs per default in such a different way feels wrong to my eyes. I guess if you implement it, it would be better to make it optional/configurable via **Settings**.

**Comment by Croydon** [comment]
@yafp I agree fully. I might implement it at same point but it will stay optional.

I haven't decided yet, since this feature is not a priority for me.

**Comment by Croydon** [comment]
It's planned, but not a priority.

**Comment by Croydon** [comment]
Blocked by #13.

**Comment by sabi0** [comment]
Another critical difference about pinned tabs in TST is that they are never scrolled off the screen.
I suppose this special panel makes that possible.

Pinned tabs are there for quick access (e.g. I have GMail as a pinned tab). And when there are many tabs open (so that a scrollbar appears) one has to scroll tab bar up and down all the time.

I would prefer the compact display for pinned tabs. I am fine to wait for that.
But would you please consider making the pinned tabs always visible with higher priority?

P.S. I know, I should cleanup my open tabs..

**Comment by Croydon** [comment]
@sabi0 That pinned tabs aren't always visible is already fixed in the new WebExtension sidebar there won't be a fix for the current legacy one. More information at #2.

**Comment by sabi0** [comment]
Thanks. Is there any way I could start using this fix already?
I couldn't find where I could install WebExtension sidebar version from.

**Comment by Croydon** [comment]
@sabi0 Sorry, for not answering in a while. At the time you asked there wasn't any satisfactory answer to your question.

Now, there is a full WebExtension version in the development channel on AMO: https://addons.mozilla.org/en-US/firefox/addon/vertical-tabs-reloaded/ (end of page)

However, it is really extremely limited right now and it is for Firefox >= 56 only, progess can be seen at #13.

So I guess it would be still better for you to wait a bit longer :)

**Comment by sabi0** [comment]
Another issue I noticed with pinned tabs is that new tabs are opened at the top of the (normal) tabs list.
For example, when I click on a link in a pinned GMail tab. I would expect such tabs to be added at the end of the tabs list.
(While middle-click in a normal tab should still open the new tab next to it)

**Comment by Croydon** [comment]
Version 0.12.0 will have minified pinned tabs as an **_experimental_ feature**.

It can be enabled in the settings:

![vtrsettings](https://user-images.githubusercontent.com/1593194/49696244-bb94c700-fba7-11e8-986b-c87f57372541.PNG)

And currently looks like this:

![vtrminifiedpinned](https://user-images.githubusercontent.com/1593194/49696260-ed0d9280-fba7-11e8-9ffb-04e872f41720.PNG)

**After 0.12.0 is released (soon), I would like to hear some feedback on this feature.** The first version where I might promote it to a stable feature is 0.13.0.

**Comment by Croydon** [comment]
I'm promoting this feature to a stable one with the 0.13.0 release.

Enabled minified pinned tabs will also become the default behaviour for 0.13.0+
